core/src/test/java/com/adobe/aem/guides/wknd/core/models/impl/ImageListImplTest.java

```diff
@@ -156,12 +156,13 @@ void SimpleImageComponentResource_getValueMap() {
 
         final ValueMap actual = new ImageListImpl.SimpleImageComponentResource(imageResource, "Test alt text").getValueMap();
 
-        assertEquals(5, actual.values().size());
+        assertEquals(8, actual.values().size());
         assertEquals("/content/dam/test.png", actual.get("fileReference"));
```

Comment:
Maybe add the test criteria to make sure that the expected lastmodified property value (and/or whatever else impacts the hash) are infact proxied through since that is the underlying fix here?